build_runner/lib/src/entrypoint/options.dart

```diff
@@ -271,7 +271,7 @@ Map<String, Map<String, dynamic>> _parseBuilderConfigOverrides(
   return builderConfigOverrides;
 }
 
-/// Returns a list of BuiltTargets with output information.
+/// Returns BuiltTargets with output information.
```

Comment:
BuildDirectory not BuiltTarget 

I also still think that something like "parsed from output arguments." would make this more clear.